components/p2p/params.go

```diff
@@ -6,8 +6,8 @@ import (
 
 // ParametersP2P contains the definition of configuration parameters used by the p2p plugin.
 type ParametersP2P struct {
-	// BindAddress defines on which address the p2p service should listen.
-	BindAddress string `default:"0.0.0.0:14666" usage:"the bind address for p2p connections"`
+	// BindAddress defines on which multi address the p2p service should listen on.
+	BindAddress string `default:"/ip4/0.0.0.0/tcp/14666" usage:"the multi address for p2p connections"`
```

Comment:
the default should bind both ipv4 and ipv6. Is this covered by this multiaddr?

Reply:
@alexsporn that can be ignored. It's fixed in my PR.